### PR TITLE
Ctcp 5537

### PR DESCRIPTION
**Pull request by Reece-Carruthers**

### CODE REVIEW

**Comment by Reece-Carruthers** [comment]
https://github.com/hmrc/manage-transit-movements-unloading-frontend/blob/779221d722f20738c3eb57f931b2247a9ca5755c/README.md

**Comment by Reece-Carruthers** [comment]
https://github.com/hmrc/manage-transit-movements-unloading-frontend/blob/51681221132be7ec5ba15ab6a306791e20b9dee6/README.md

File: README.md [hunk 1]
```diff
@@ -8,23 +8,47 @@ Service manager port: 10123
...
 <pre>sbt A11y/test</pre>
 
 ### Running manually or for journey tests
 
+To toggle between the Phase 5 transition and post-transition modes we have defined two separate modules, each with their own set of class bindings to handle the rules associated with these two periods.
+
+#### Transition
+<pre>
+sm2 --start CTC_TRADERS_P5_ACCEPTANCE_TRANSITION
+sm2 --stop MANAGE_TRANSIT_MOVEMENTS_UNLOADING_FRONTEND
```
Comment (jackheslop96) [inline README.md:24]: Should be `MANAGE_TRANSIT_MOVEMENTS_UNLOADING_FRONTEND_TRANSITION`

Comment (Reece-Carruthers) [reply README.md]: Fixed

File: README.md [hunk 2]
```diff
@@ -8,23 +8,47 @@ Service manager port: 10123
...
+
+#### Transition
+<pre>
+sm2 --start CTC_TRADERS_P5_ACCEPTANCE_TRANSITION
+sm2 --stop MANAGE_TRANSIT_MOVEMENTS_UNLOADING_FRONTEND_TRANSITION
+sbt -Dplay.additional.module=config.TransitionModule run
+</pre>
+
+#### Final
+Phase 5 final introduces multi-house consingment allowing traders to enter multiple house consignments instead of a singular one.
```
Comment (jackheslop96) [inline README.md:29]: Bit waffly 😆 Can we say something like "Phase 5 final introduces multiple house consignments. This provides users with functionalities that are not available during the transition period, like the ability to add and remove entire house consignments."

Comment (Reece-Carruthers) [reply README.md:29]: Just a bit 😆, that sounds a lot better. I will put that in

**Review by jackheslop96 — APPROVED** [review]

**Pull request merged**